package.json

```diff
@@ -84,63 +67,54 @@
     "classnames": "^2.2.5",
     "css-loader": "^3.6.0",
     "cypress": "^6.5.0",
-    "cypress-plugin-tab": "^1.0.1",
+    "cypress-plugin-tab": "^1.0.5",
     "dotenv-cli": "^3.2.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.1",
     "es-abstract": "^1.16.0",
-    "eslint": "^7.20.0",
-    "eslint-config-prettier": "^6.9.0",
-    "eslint-config-react-app": "^2.1.0",
-    "eslint-config-sparkpost": "^1.4.1",
-    "eslint-plugin-flowtype": "^2.46.3",
-    "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-jest": "^21.15.1",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-lodash": "^2.7.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "^7.8.2",
-    "husky": "^0.14.3",
+    "eslint": "^8.18.0",
+    "eslint-config-next": "^12.1.6",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.30.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "husky": "^8.0.1",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "jest-enzyme": "^7.1.2",
+    "jest-in-case": "^1.0.2",
     "jest-styled-components": "^7.0.3",
-    "lerna": "^4.0.0",
-    "lerna-changelog": "^1.0.1",
     "lint-staged": "^7.1.0",
-    "lodash.memoize": "^4.1.2",
     "plugin-error": "^1.0.1",
-    "postcss-loader": "^3.0.0",
-    "prettier": "^2.3.2",
-    "react-test-renderer": "^16.8.6",
+    "prettier": "^2.6.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-transition-group": "^4.3.0",
     "resize-observer-polyfill": "^1.5.1",
     "rimraf": "^3.0.2",
     "sass": "^1.49.0",
-    "sass-loader": "^12.4.0",
-    "serve": "^11.3.2",
+    "sass-loader": "^12.6.0",
+    "serve": "^13.0.2",
     "start-server-and-test": "^1.11.7",
     "style-loader": "^1.3.0",
-    "stylelint": "^13.13.1",
-    "stylelint-config-prettier": "^8.0.2",
-    "stylelint-config-recommended": "^3.0.0",
-    "stylelint-config-styled-components": "^0.1.1",
-    "stylelint-prettier": "^1.1.2",
-    "stylelint-processor-styled-components": "^1.10.0",
-    "theo": "^8.1.5",
-    "ts-loader": "^9.2.5",
     "tslib": "^2.3.0",
     "typescript": "^4.3.5"
   },
-  "changelog": {
-    "cacheDir": ".changelog",
-    "labels": {
-      "type: feature": "Features",
-      "type: enhancement": "Enhancements",
-      "type: bug": "Bug Fixes",
-      "type: breaking change": "Breaking Changes",
-      "type: security": "Security",
-      "type: infrastructure": "Infrastructure"
-    }
+  "overrides": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "workspaces": [
+    "packages/design-tokens",
+    "packages/matchbox-css",
+    "packages/matchbox-icons",
+    "packages/matchbox-media",
+    "packages/matchbox"
```

Comment:
Intentionally left out the cra and next templates since they are a bit unique and don't have a standard package structure